protos/schedules.proto

```diff
@@ -2,12 +2,42 @@ syntax="proto3";
 
 package dataform.schedules;
 
+enum NotificationEventType {
+  ON_ANY = 0;
```

Comment:
i don't think we need this value (it corresponds to a non-existent `events` list IMO)

Reply:
I get error, that Enum should have a index of 0, what should I do about that?

Reply:
Do we need this enum at all?
If so, I'd just call the first value - UKNOWN (that's what we do everywhere else).